.semaphore/semaphore.yml

```diff
@@ -12,9 +12,9 @@ blocks:
         - name: Get dependencies.
           commands:
              - checkout
-             - sudo apt-get -y update
+             - sudo apt -y update
              - sudo apt-get -y install cmake gettext help2man clang gcc-7 g++-7 libepoxy-dev libsdl2-dev libcairo2-dev libpango1.0-dev librsvg2-dev libboost-all-dev libavcodec-dev libavformat-dev libswscale-dev libswresample-dev libpng-dev libjpeg-dev libxml++2.6-dev portaudio19-dev libopencv-dev libportmidi-dev libjsoncpp-dev libqrencode-dev libicu-dev libcpprest-dev libglm-dev
```

Comment:
Also use `sudo apt install -y package-1 package-2 package-3`

Reply:
Will change it later. Anyway this was made to try and get around the insufficient space issue which is completely a problem on semaphore's side.

Reply:
I thought so hahah. Hmm is there an issue or such which we can follow?

Reply:
I've tracked the issue down to libsdl2, which requires udev and that in turn wants a new kernel, which causes the error with initramfs-tools. Still, that would be solved by making the /boot partition larger. I've made a thread on the semaphore Slack so if you've got Slack you can follow it there.